client/src/components/AddQuestionForm.tsx

```diff
@@ -0,0 +1,166 @@
+      }
+    }
+  };
+
+  interface AnswerSendProp {
+    [key: string]: number;
+  }
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    const filteredData = {
+      ...data,
+      answers: data.answers.filter(
+        (answer) =>
+          answer.answer.trim() !== "" && answer.numberOfResponse !== null
+      ),
+    };
+
+    const totalResponses = filteredData.answers.reduce(
+      (acc, answer) => acc + +answer.numberOfResponse,
+      0
+    );
+    if (totalResponses !== 100) {
+      console.error("Total responses must be 100");
+      setCorrectScore(false);
+      return;
+    }
+
+    const answersToSend: AnswerSendProp = {};
+    filteredData.answers.forEach((answer) => {
+      answersToSend[answer.answer] = +answer.numberOfResponse;
+    });
+
+    const sendObject = {
+      question: filteredData.question,
+      answers: answersToSend,
+    };
+
+    try {
+      await adminQuickRound.createQuestion(sendObject);
+
+      queryClient.invalidateQueries({ queryKey: ["questions"] });
+      setCorrectScore(true);
+      closeModal();
+    } catch (error) {
+      console.error("Failed to create question:", error);
+    }
+  };
+
+  if (isSubmitting) return <Spinner />;
+
+  return (
+    <div className="flex justify-center items-center h-auto w-full">
+      <div className="p-12 bg-opacity-60 w-full max-w-3xl mx-auto">
+        <div className="mb-6">
+          <label className="block text-gray-50 mb-2">Number of Answers</label>
+          <span className="text-red-500">
+            {correctScore ? "" : "Total Points must be 100"}
+          </span>
+          <input
+            type="number"
+            min="1"
+            max="10"
+            value={numOfAnswers}
+            onChange={handleAnswerChange}
+            className="block w-full max-w-md bg-transparent outline-none border-b-2 border-black py-2 px-4 text-gray-50"
+            placeholder="Enter number of answers"
+          />
+        </div>
+
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          className="w-full max-w-3xl mx-auto"
+        >
+          <FormInputField
+            type="text"
+            register={register("question", {
+              required: "Question is required",
+            })}
+            label="Question"
+            error={errors.question}
+            placeholder="Enter your question"
+          />
+
+          {[...Array(numOfAnswers)].map((_, index) => (
+            <div key={index} className="flex gap-4 mb-6">
+              <FormInputField
+                type="text"
+                register={register(`answers.${index}.answer`, {
+                  required: "Answer is required",
+                })}
+                label={`Answer ${index + 1}`}
+                error={errors.answers?.[index]?.answer}
+                placeholder="Enter answer"
+                className="flex-1"
+              />
+              <FormInputField
+                type="number"
```

Comment:
On all number type inputs, remove the ability to change the value by scrolling
It is impractical when the user has to input multiple numbers
See image for clarification
https://imgur.com/a/gqg2TMO